.circleci/config.yml

```diff
@@ -212,6 +220,16 @@ workflows:
                 - env/dev2
             tags:
               only: /^v.*$/
+      - deploy_api_docs:
+          requires: ~
+          requires:
```

Comment:
I don't think you need the build? We probably need `static_analysis` requirement to ensure the swagger docs are in sync before publishing cc @lucafavatella

Reply:
Indeed depending on swagger-check would help

Reply:
Yep, but we don't have separate job for it, so requirement should go to `static_analysis` however it implicitly requires `build`